discord/http.py

```diff
@@ -428,6 +429,9 @@ def send_message(
 
         if embed:
             payload['embed'] = embed
```

Comment:
```suggestion
        if embed:
            payload['embeds'] = [embed]
```
Changelog says
> The existing embed field is considered deprecated and will be removed in the next API version.
so embed should use embeds field.

Reply:
The  embed field isn't delete yet so I haven't made change like you.
When the embed field were deleted from API (=API version were updated), me or someone will make change like you.
(But maybe someone other than me will make a pull request like your change and it will be merged.)

Reply:
There is no point not to change it now, so you should do it

Reply:
That's for sure. I fixed it.